src/components/interfaces/MOBILE_API.xml

```diff
@@ -4971,86 +5139,6 @@
     </param>
   </function>
 
-  <function name="SubscribeWayPoints" functionID="SubscribeWayPointsID" messagetype="request">

```

Comment:
@LuxoftAKutsan reverted in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/ab9adc7c6996e68e910941d6e3ed04f751e49215